js/main.js

```diff
@@ -153,3 +168,31 @@ $profileButton.setAttribute('data-view', 'profile');
 $profileButton.id = 'profile-button';
 $profileButton.textContent = 'Profile'
 $header.appendChild($profileButton);
+
+var $entriesButton = document.createElement('button');
+$entries.setAttribute('href', '#');
+$entriesButton.setAttribute('data-view', 'entries');
+$entriesButton.id = 'entries-button';
+$entriesButton.textContent = 'Entries'
+$header.appendChild($entriesButton);
+
+var $createEntryButton = document.createElement('button');
+$createEntryButton.setAttribute('href', '#');
+$createEntryButton.setAttribute('data-view', 'create-entry');
+$createEntryButton.id = 'create-entry-button';
+$createEntryButton.textContent = 'NEW'
+$entries.appendChild($createEntryButton);
+
+$imgUrl.addEventListener('input', function(event){
+  $entryPicture.src = event.target.value;
+})
+
+$entryForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+  newEntry['imgUrl'] = $imgUrl.value;
+  newEntry['entryTitle'] = $entryTitle.value;
+  newEntry['entryNotes'] = $entryNotes.value;
```

Comment:
You seem to be storing the newEntry object in the global scope and then overwriting it when there is a new entry.  It is better practice to create an object and store it in your data model.  If you choose not to fix this now, you will in the next feature.

```
var entry = {
  imgUrl: $imgUrl.value,
  title: $entryTitle.value,
  notes: $entryNotes.value
}
data.entries.push(entry)

Reply:
![image](https://user-images.githubusercontent.com/73401059/101226656-0d16a600-364a-11eb-9a09-d354eb664097.png)